program/src/stake_account.rs

```diff
@@ -196,6 +203,8 @@ impl StakeAccount {
 
         // This toggle is a historical quirk in Solana and should always be set
         // to true. See also https://github.com/ChorusOne/solido/issues/184#issuecomment-861653316.
+        // Note: this has all finally been removed in 1.7, so you can get rid of it
+        // whenever all of the code is updated to the newest SDK.
```

Comment:
I’t still there in 1.7.10, but it’s gone in 1.7.11 ... do you do breaking changes in patch releases? :fearful: 

We are building against 1.7.8 and stuck with it for now because the multisig program uses Anchor which pins Solana to 1.7.8. But good to know, and glad to hear it’s gone! I guess will find out automatically when we upgrade.

Reply:
> do you do breaking changes in patch releases?

Sometimes... sorry. It'll be an easy fix at least!